### PR TITLE
Only removed unused constants.

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**
Removed unused constants.

### CODE REVIEW

File: src/constants/label.js [hunk 1]
```diff
@@ -1,41 +1,5 @@
...
-
-export const EMAIL_LABEL = 'ई-मेल';
-
-export const ADDRESS_LABEL = 'पूरा पता';
-
-export const BUS_STOP_LABEL = 'बस स्टॉप (कृपया निकटतम बस स्टॉप चुनें)';
-
-export const WHAT_YOU_WANT_TO_STUDY_LABEL = 'आप क्या अध्ययन करना चाहते हैं ?';
-
 export const MARK_SELECTED_STUDENTS_ATTENDANCE_LABEL = 'कृपिया चयनित छात्रो की उपस्तिथि दर्ज करे ';
```
Comment (Nikmahajan) [inline src/constants/label.js]: Why this constant is named student?

File: src/constants/yjsg.js [hunk 2]
```diff
@@ -1,246 +1,7 @@
...
-  },
-  {
-    text: 'mobile',
-    value: 'mobile',
-  },
-];
-
-export const yjsgHeader = 'जैन बाल एवं युवा संस्कार शिक्षण शिविर (चतुर्थ वर्ष)';
-
 export const yjsgFooterText = 'किसी भी अन्य जानकारी प्राप्त करने हेतु संपर्क सूत्र:';
```
Comment (Nikmahajan) [inline src/constants/yjsg.js]: No hard coding.

File: src/constants/yjsg.js [hunk 3]
```diff
@@ -273,271 +34,10 @@ export const infoUpdateSuccessMessage = 'आपकी जानकारी स
...
-    value: 'B26',
-  },
-  {
-    text: 'B27',
-    value: 'B27',
-  },
-];
-
 export const USER_TYPES = {
   STUDENT: 'student',
```
Comment (Nikmahajan) [inline src/constants/yjsg.js]: Student user is not generic.

File: src/constants/label.js (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/constants/label.js]: Done.

File: src/constants/yjsg.js (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/constants/yjsg.js]: This can refactor in the future as for default route.

File: src/constants/yjsg.js (hunk 3 above)
Comment (BhagyashreeBhoyar) [reply src/constants/yjsg.js]: Done.

File: src/constants/yjsg.js (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/constants/yjsg.js]: Done.

**Pull request merged**